src/app/zones/locales/en-us/index.yaml

```diff
@@ -1,29 +1,47 @@
 zone-cps:
   routes:
     item:
-      title: "{name} Zone CP"
-      breadcrumbs: Zone CPs
+      title: "{name} Control Plane"
+      breadcrumbs: Control Planes
       config:
         no-subscriptions: This zone has no subscriptions
     items:
-      title: Zone CPs
-      breadcrumbs: Zone CPs
+      title: Control Planes
+      breadcrumbs: Control Planes
+      headers:
+        name: 'Name'
+        zoneCpVersion: 'Zone CP Version'
+        storeType: 'Storage type'
+        status: 'Status'
+        warnings: 'Warnings'
+        actions: 'Actions'
+
 zone-ingresses:
   routes:
     item:
       title: "{name} Zone Ingress"
-      breadcrumbs: Zone Ingresses
+      breadcrumbs: Ingresses
     items:
-      title: Zone Ingresses
-      breadcrumbs: Zone Ingresses
+      title: Ingresses
+      breadcrumbs: Ingresses
+      headers:
+        name: 'Name'
+        status: 'Status'
+        actions: 'Actions'
```

Comment:
As above re: `actions`

Reply:
Not moving this. Table headers should come from one place even though this might be duplicated.